frontend/packages/console-app/src/components/quick-starts/QuickStartController.tsx

```diff
@@ -6,12 +6,14 @@ import { QuickStartContext, QuickStartContextValues } from './utils/quick-start-
 
 type QuickStartControllerProps = {
   quickStart: QuickStart;
+  nextQuickStarts?: QuickStart[];
```

Comment:
do we need this to be a list or just `QuickStart` ?

Reply:
A quickStart could have multple `nextQuickStarts` as dependencies. Although we only make use of the first one as of today. But, lets not lose this data for future needs.

Reply:
We should show all the next quick starts. Can you make that change and just loop over all the next quick starts to render all of them?

Reply:
💯 agree... if we have it we should show it. Maybe that's a 4.8 thing and not part of this bug though.

Reply:
PR is ready for re-review with single `nextQuickStart` @rohitkrai03 @invincibleJai